integration_tests/features/WalletCli.feature

```diff
@@ -152,26 +152,26 @@ Feature: Wallet CLI
         And mining node MINE mines 1 blocks
         Then WALLET is connected to BASE
 
-    @dan_layer
+    @dan @critical
     Scenario: As a user I want to publish a contract definition via command line
         Given I have a base node BASE
         And I have wallet WALLET connected to base node BASE
         And I have mining node MINE connected to base node BASE and wallet WALLET
         And mining node MINE mines 4 blocks
         Then I wait for wallet WALLET to have at least 1000000 uT
-        And I create a "contract-definition" from file "fixtures/contract_definition.json" on wallet WALLET via command line
-        And mining node MINE mines 4 blocks
+        And I publish a contract definition from file "fixtures/contract_definition.json" on wallet WALLET via command line
+        And mining node MINE mines 8 blocks
         Then wallet WALLET has at least 1 transactions that are all TRANSACTION_STATUS_MINED_CONFIRMED and not cancelled
         Then WALLET is connected to BASE
 
-    @dan_layer
-    Scenario: As a user I want to publish a constitution definition via command line
+    @dan @critical
+    Scenario: As a user I want to publish a contract constitution via command line
         Given I have a base node BASE
         And I have wallet WALLET connected to base node BASE
         And I have mining node MINE connected to base node BASE and wallet WALLET
         And mining node MINE mines 4 blocks
         Then I wait for wallet WALLET to have at least 1000000 uT
-        And I create a "constitution-definition" from file "fixtures/constitution_definition.json" on wallet WALLET via command line
-        And mining node MINE mines 4 blocks
+        And I publish a contract constitution from file "fixtures/contract_constitution.json" on wallet WALLET via command line
```

Comment:
The contract definition must exist before the constitution is published. In future this test will break, so I think you'll need to add a contract definition first in this test.

Reply:
This turns out to cause issues, resolving this because it can be fixed in another PR

Reply:
I agree. Seems like by publishing both the definition and constitution in the same test, the definition UTXO gets picked as an input for the constitution UTXO, and we don't want that. As the fix is a bit delicate, we can isolate it into a separated PR.